pallets/gear-program/src/migrations/add_section_sizes.rs

```diff
@@ -0,0 +1,262 @@
+// This file is part of Gear.
```

Comment:
(!) To be reviewed once decided on code data structures

Reply:
I don't get it. What do you suggest here?